### PR TITLE
Feature/download pdf

### PR DESCRIPTION
**Pull request by ali-7**
* Downloadable PDF Summary 

relates #160

### CODE REVIEW

**Review by teenie-quaggard — CHANGES_REQUESTED** [review]
happy for this to be merged after these little things are checked out: 
Screen Shot 2019-05-05 at 08 56 54

don't wait for a second review, as i'll be en route to see david! i trust you :)
Thank you!!!
looks great.

**Review by hshahwan — APPROVED** [review]
Great job Ali

**Comment by ali-7** [comment]
@teenie-quaggard first one form the browser, second there's new prop added in the method cards, no relation to this pr :joy: , anyway fixed

**Pull request merged**
